pkg/epforwarder/epforwarder.go

```diff
@@ -98,9 +98,20 @@ var passthroughPipelineDescs = []passthroughPipelineDesc{
 		hostnameEndpointPrefix:        "ndmflow-intake.",
 		intakeTrackType:               "ndmflow",
 		defaultBatchMaxConcurrentSend: 10,
-		defaultBatchMaxContentSize:    20e6, // max 20Mb uncompressed size per payload
-		defaultBatchMaxSize:           pkgconfig.DefaultBatchMaxSize,
-		defaultInputChanSize:          pkgconfig.DefaultInputChanSize,
+		defaultBatchMaxContentSize:    pkgconfig.DefaultBatchMaxContentSize,
```

Comment:
What's the default coming from `pkgconfig` here?

Reply:
It's 5Mb.